src/GameScore/GameScore.cpp

```diff
@@ -7,258 +6,231 @@
 
 GameScore::GameScore() {}
 
-GameScore::~GameScore() {
-
-}
+GameScore::~GameScore() {}
```

Comment:
Could we define this or remove it entirely? If there's nothing to clean up I think we can just the one provided by the compiler.

Reply:
Yup, good call. Removed!